README.md

```diff
@@ -297,6 +297,24 @@ match change_password
 
 You'll be asked for the new password on all your machines on the next run.
 
+### Manual control
+
+If you for any reason do not have access to the Dev portal, you can still save your certificates and profiles in match.
+
+To save your certificate, run.
+```
+match manual -p path/to/key.p12 -c path/to/my.mobileprovision -r https://github.com/fastlane/certificates -a tools.fastlane.app -y appstore
```

Comment:
You might be able to find the known identities in the keychain, but matching certificates up against the correct mobileprovision may prove troublesome... or at least I do not know how to do that.